renovate.json

```diff
@@ -9,11 +9,24 @@
   "customManagers": [
     {
       "customType": "regex",
-      "fileMatch": ["^.github/test_dependencies.yaml$"],
+      "fileMatch": [
+        "^.github/test_dependencies.yaml$"
+      ],
       "matchStrings": [
         "#\\s+renovate:\\s+datasource=(?<datasource>.*)\\s+depName=(?<depName>.*)\\s+versioning=(?<versioning>.*)\\n.+'(?<currentValue>.*)'"
       ]
+    },
+    {
+      "customType": "regex",
+      "fileMatch": [
+        "^test/consts/helm.go$"
```

Comment:
Do you think we could extract the definition of the version used in tests to `test_dependencies.yaml` (under `conformance.kongHelmChart` and `integration.kongHelmChart`)? It would require adding `TEST_KONG_HELM_CHART_VERSION` to inject that and a test helper function to read it.

Reply:
~I thought about it but then decided that I don't want to over-engineer it for now. The test helper made me not do it. We don't have a "framework" for reading the `.github/test_dependencies.yaml` file do we? If we had that would have been easier.~

Let me try that:  https://github.com/Kong/kubernetes-ingress-controller/pull/4846/commits/3c7f1d852158e07ee71d74d8b6e995bb05297fce